### PR TITLE
[ADD] sales: create quotations doc

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/odoo/3835/tasks/3981941?cids=3

Surprisingly, there is no documentation to inform Odoo users how to properly create (and send) quotations using the Sales application. There is a doc that focuses briefly on how to 'Create and send quotations' with the CRM application, via a qualified lead/opportunity -- but, there is no doc for the Sales app that properly focuses on the quotation form, how to fill it out, and how to send it. That's why this particular doc is being written. Also, it will include a reference to the new 'Catalog' option that is present in the Order Lines tab, allowing users to freely peruse the product catalog in the database to add products to a quotation (as suggested to me by VPD to include).

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Alrighty, @hojo-odoo -- this one is ready for a Peer Review whenever you get a chance. Thanks!

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:191]:
```suggestion
four different sections: :guilabel:`Sales`, :guilabel:`Delivery`, :guilabel:`Invoicing`, and
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:220]:
```suggestion
  entered reference ID can contain letters, numbers, or a mix of both.
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:227]:
```suggestion
In the :guilabel:`Delivery` section of the :guilabel:`Other Info` tab, there are delivery-specific
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:237]:
```suggestion
  commerical terms for international transactions.
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:241]:
```suggestion
  products are delivered at once, the delivery order is scheduled based on the greatest product
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:275]:
```suggestion
- :guilabel:`Source Document`: enter the reference of the document that generated the
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:280]:
```suggestion
- :guilabel:`Medium`: select the method by which this quotation originated (e.g. *Email*,
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:308]: "Proforma" is formatted differently here than above.

**Review by hojo-odoo — APPROVED** [review]
Hey @ksc-odoo,

Great job on this. It should be ready for the next round of review once you take a look at my comments.

Thanks,
John

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:58]: This is a great question. And, frankly, I am not entirely certain. Just to explain my stance, I feel like, since I'm **only** mentioning *if* they are in the application (and not directing them to navigate to/open the app), I opted for the use of italics, instead of the menuselection.

However, I fully see your point, and understand this a bit of an "unorthodox" way to format such a sentence/direction.

So, since this is a kind of "up in the air" issue, I am going to choose to leave it as-is for now, but if ZST or SALI think a different approach should be taken, I'll be sure to make the necessary adjustment. Thanks for bringing this to my attention, though. Great eye for detail 👁️ 👍

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:308]: For sure, and I would've mentioned the same thing. However, like some other aspects of the U/I, the spelling/formatting isn't always the same throughout the database, unfortunately. So, with that in mind, I always opt for the formatting/spelling that is present on the screen *at the moment*.

In this case, it is spelled/formatted like this one the pop-up:
Screenshot 2024-06-13 105142

**Comment by ksc-odoo** [comment]
Thanks for the helpful feedback, @hojo-odoo -- I made all the suggested edits, and believe this is now ready for a Final Review, @StraubCreative @odoo/us-doc-review

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:122]: you could make an anchor to the settings and link it here- totally optional!

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:149]:
```suggestion
Or, click :guilabel:`Catalog` to reveal a separate page, showcasing every item, and every potential
product variant, in a neatly organized display, with items organizable by :guilabel:`Product
```

Comment (larm-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:149]: Also, I am not sure if I am correct- but it appears to me that these are organized alphabetically by the Internal Reference code on the product form. I checked in a runbot, and it seems to be the case. You can filter with categories and attributes, but I do not think it is actually organizing them in that order?

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:157]:
```suggestion
button on the product cards, and adjust the quantity, if needed. When complete, click the
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:156]: You don't have to click the cart- clicking anywhere on the card adds more items. This confused me since I clicked on a card to reveal the product details, and it added it to the cart!  Kind of odd to even have a cart button at that point =D
Just wanted to mention it!

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:163]: You could add an example here- I know it's probably unnecessary, but perhaps to illustrate the ways to divide products. You could do a clothing company, and have it organized by Women, Men's, Kids, etc. Or perhaps a tech company, with items, and  section for accessories? Just a thought!

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:166]: Looks like the note can only be at the bottom- so it doesn't appear under whatever product the note is needed for.  So if you need to add a note to a product, you have to add it, make sure it's at the bottom- add the note, then add more products, etc. I tested this a few times and it seems to be how it works. I'd add an "important' admonition or something similar since this could e a huge issue if someone adds a lot of products, thinking they can add notes after they are added.

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:208]: Is the person in this field the one who receives a commission? Might be good to note.

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:234]:
```suggestion
- :guilabel:`Shipping Weight`: the weight of the items being shipped. This field is not
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:262]: There's a confirmation window that appears as well.

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (larm-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:275]:
```suggestion
- :guilabel:`Source Document`: enter the reference of the document that generated the
```

**Review by larm-odoo — APPROVED** [review]
Hi @ksc-odoo - another great doc! I just had some suggestions for some additional information here and there. Approving now!

**Comment by StraubCreative** [comment]
@ksc-odoo to merge this in please address @larm-odoo feedback and pass to @samueljlieber after, thanks!

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:38]: It is a warning message that is sent to the user. I added a mention about this in the explanation. 👍 
Screenshot 2024-07-16 092738

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:46]: I'm going to keep this formatting as-is, because I don't want the reader to feel like they need to select more than one option. But I completely understand (and appreciate) this suggestion 😄

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:111]: Yes. This is for commission. I feel like I mention that in a few different ways here. I'll try to think of a way to make it even more clear, but I'm not sure if that's possible. That said, I think I'm going to keep this as-is, as well.

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:149]: When I mention that it is "organizable" that was intended to imply that users can *choose* to organize the results on the catalog page using the sidebar options. Maybe you mistakenly read that as "**is** organized by..." 🤷

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:156]: Oooh that *is* interesting. Great catch! I'm assuming this **might** be a bug that is fixed later...? Not sure. Either way, I think if I keep this as-is, I won't have to return to it down the line to specify that the 'Add' button needs to be clicked. At this point, though, like you said, clicking anywhere on the card adds it, so users may see that as a "happy accident" haha

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:166]: Well, actually, when you write a note, it appears as a line on the Order Lines tab (just like a product or section). And, just like a product or section line, they can be moved and placed anywhere on the quotation. I added some more context about that functionality. 👍

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:208]: Well, that's a bit tricky. So, it kinda depends on how the individual company operates. There could be a referrer who gets a commission. Or, the commission could go to the salesperson. Or, the company could divvy such a thing up between the two (if both are even involved). So, considering that it's a bit of a "case by case" basis, I am going to leave it as this generic explanation, so as to avoid any confusion.

**Comment by ksc-odoo** [comment]
Thanks for all the helpful feedback @larm-odoo -- I made the necessary suggested edits, and believe this one is now ready for Tech Review, @samueljlieber (whenever you get a chance). Thanks! 👍

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:113]: early line break
```suggestion
If a :guilabel:`Referrer` is selected, a new field, :guilabel:`Commission Plan` appears, in which a
commission can be selected from the drop-down menu. This commission is rewarded to the contact
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:248]: early line break
```suggestion
  time. Otherwise, it is based on the shortest lead time. The available options are: :guilabel:`As
  soon as possible` or :guilabel:`When all products are ready`.
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:250]: early line break
```suggestion
- :guilabel:`Delivery Date`: click into the empty field to reveal a calendar popover, from which a
  customer delivery date can be selected. If no custom date is required, refer to the
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:24]: All of the "to learn more" items in this list can just be accumulated in a **seealso** admonition at the end, or the links can be more integrated with the list item descriptions as inline links or individual **seealso**s; we don't need to say over and over "to learn more go to the x documentation"

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:37]: redundant "default"
```suggestion
- :guilabel:`Default Quotation Validity`: determine a set amount (in :guilabel:`days`) that quotations
  can remain valid for.
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:61]: Sentence too long to say a simple/obvious thing.
The tip isn't necessary.
Why 11 lines for such a basic navigational instruction?

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

See the necessary technical changes below. Once these are addressed we can merge.

Also for your consideration, I've highlighted a few content-related items. 

Overall, I'd like to see the written content be much more concise. There are a number of spots where I (the reader) am being forced to read long or multiple sentences for super basic or obvious things. On this doc and other content please try to cut down on the verboseness and keep the tone direct, instructional, and assertive.

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:24]: Understood. Moved them to the seealso at the end of the doc. Thanks for the super helpful suggestion. I really appreciate the attention to detail. 🙏

**Comment by ksc-odoo** [comment]
Hey @StraubCreative -- first off, thank you SO MUCH for the incredibly thoughtful, super-detailed feedback. I truly appreciate it, and believe it continues to improve my writing. So, thank you for taking the time to provide me with such useful comments.

As you can imagine, this PR was created far before we discussed my new objective to be more concise, so that explains the need to "whittle things down" in this PR. That said, I **fully** agree with your directives and feedback, and have made the necessary adjustments suggested by you on this PR, *and* I made the required modifications (formatting-wise) to fall in line with the *brand new* RST guidelines you provided the team on Discord. 

So, whenever you get a free moment, if you could give this another quick look to see if things are ready to go here, that would be awesome. And, if there are any additional adjustments you'd like me to make, just let me know, and I'll get it done right away.

Thanks again! 🙏

File: content/applications/sales/sales/send_quotations/create_quotations.rst
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:272]: Specific platforms like *Facebook* are actually **Source** values. *Email* is an acceptable **Medium** though. 

[Here's a doc](https://www.linkedin.com/pulse/whats-difference-between-source-medium-utm-parameters-/) that does a good job covering the difference between the two.

**Source** examples: facebook, google, bing, quora, youtube, reddit, etc.
**Medium** examples: cpc, organic-social, paid-social, email, email, etc.

Comment (StraubCreative) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:272]: This section might be a good opp to do a **seealso** or a **tip** to use the link tracker app which has a form to generate UTM tracking values.

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 

I think this doc came out nicely, good job on the revisions and especially on being more concise. This thing is sharp!

I only had one small comment regarding the UTM values in the **Tracking section**. See below and merge when you're ready 👍 
.....
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
